txtorcon/torinfo.py

```diff
@@ -99,6 +109,9 @@ def stripper(key, arg):
 
         d = self.proto.get_info_raw(req)
         d.addCallback(functools.partial(stripper, req))
+        descriptor_keys = ["desc/name", "desc/id"]
+        if self.info_key in descriptor_keys and self.proto.use_stem:
+            d.addCallback(_parse_stem_descriptor)
         return d
```

Comment:
I think its better to have the RelayDescriptor constructor as the callback. This is because all the function is doing is just calling RelayDescriptor(data). The changes will be seen in the next commit.